controllers/nodeagent.go

```diff
@@ -133,49 +156,57 @@ func (r *DPAReconciler) ReconcileResticDaemonset(log logr.Logger) (bool, error)
 			if isStatusCause && cause.Field == "spec.selector" {
 				// recreate deployment
 				// TODO: check for in-progress backup/restore to wait for it to finish
-				log.Info("Found immutable selector from previous daemonset, recreating restic daemonset")
+				log.Info("Found immutable selector from previous daemonset, recreating NodeAgent daemonset")
 				err := r.Delete(r.Context, ds)
 				if err != nil {
 					return false, err
 				}
-				return r.ReconcileResticDaemonset(log)
+				return r.ReconcileNodeAgentDaemonset(log)
 			}
 		}
 		return false, err
 	}
 
 	if op == controllerutil.OperationResultCreated || op == controllerutil.OperationResultUpdated {
-		// Trigger event to indicate restic was created or updated
+		// Trigger event to indicate NodeAgent was created or updated
 		r.EventRecorder.Event(ds,
 			corev1.EventTypeNormal,
-			"ResticDaemonsetReconciled",
-			fmt.Sprintf("performed %s on restic deployment %s/%s", op, ds.Namespace, ds.Name),
+			"NodeAgentDaemonsetReconciled",
+			fmt.Sprintf("performed %s on NodeAgent deployment %s/%s", op, ds.Namespace, ds.Name),
 		)
 	}
 
 	return true, nil
 }
 
 /**
- * This function builds restic Daemonset. It calls /pkg/credentials function AppendCloudProviderVolumes
+ * This function builds NodeAgent Daemonset. It calls /pkg/credentials function AppendCloudProviderVolumes
  * args: velero - the velero object pointer
  * 		 ds		- pointer to daemonset with objectMeta defined
  * returns: (pointer to daemonset, nil) if successful
  */
-func (r *DPAReconciler) buildResticDaemonset(dpa *oadpv1alpha1.DataProtectionApplication, ds *appsv1.DaemonSet) (*appsv1.DaemonSet, error) {
+func (r *DPAReconciler) buildNodeAgentDaemonset(dpa *oadpv1alpha1.DataProtectionApplication, ds *appsv1.DaemonSet) (*appsv1.DaemonSet, error) {
 	if dpa == nil {
 		return nil, fmt.Errorf("dpa cannot be nil")
 	}
 	if ds == nil {
 		return nil, fmt.Errorf("ds cannot be nil")
 	}
 
+	var nodeAgentResourceReqs corev1.ResourceRequirements
+
 	// get resource requirements for restic ds
```

Comment:
NIT: // get resource requirements for nodeAgent ds

Reply:
Changed.